frontend/src/app/auth-guards/is-admin-auth-guard.ts

```diff
@@ -1,11 +1,11 @@
 import { Injectable } from '@angular/core';
-import { CanActivate, Router } from '@angular/router';
+import { Router } from '@angular/router';
```

Comment:
please revert, CanActivate is still available in Angular 18+

https://github.com/angular/angular/pull/56408